packages/block-library/src/embed/embed-placeholder.native.js

```diff
@@ -8,13 +8,24 @@ import { View, Text, TouchableWithoutFeedback } from 'react-native';
  */
 import { __ } from '@wordpress/i18n';
 import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+import { useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
+import EmbedBottomSheet from './embed-bottom-sheet';
 import styles from './styles.scss';
 
-const EmbedPlaceholder = ( { icon, label, onFocus } ) => {
+const EmbedPlaceholder = ( {
+	icon,
+	label,
+	onFocus,
+	value,
+	onSubmit,
+	onChange,
+} ) => {
+	const [ isEmbedSheetVisible, setIsEmbedSheetVisible ] = useState( ! value );
```

Comment:
Noticed a flow that triggers the bottom sheet to show: If you add the block in html mode (i.e. add a `<!-- wp:embed /-->` markup) and switch to visual mode, the bottom sheet gets opened automatically. Can we make the flow specific, to only auto-open when added in visual mode?

Reply:
Good catch!

> Can we make the flow specific, to only auto-open when added in visual mode?

Yeah, we could have a similar flow as we have for the media blocks that only in case the component is inserter from the inserter menu we displayed the bottom sheet.

Reply:
Addressed with https://github.com/WordPress/gutenberg/pull/31343/commits/0d704574ed2bd6e811628602d58b315af7f30391.